### PR TITLE
Debugdemo

### PR DESCRIPTION
**Pull request by Cindie1908**

### CODE REVIEW

File: back/routes/bookings.js [hunk 1]
```diff
@@ -120,9 +120,10 @@ router.post('/', (request, response) => {
 });
 
 
-router.delete('/tourist/:id', (req, res) => {
+router.delete('/tourist/:id/:id', (req, res) => {
   const idVacationer = req.params.id;
-  connection.query('DELETE FROM booking WHERE tourist_id = ?', [idVacationer], err => {
+  const idEvent = req.params.id;
```
Comment (ghost) [inline back/routes/bookings.js]: I think req.params.id and req.params.id are the same, give them different names in route path

File: front/src/component/DeletionOfBookingsByTourist.js [hunk 2]
```diff
@@ -1,105 +1,30 @@
...
-    <button id="buttonDelete" type="button" onClick={() => deleteFormData(id)} type='submit' value='Supprimer'> </button>  
+    <button id="buttonEye" type="button" onClick={event=> { event.preventDefault(); history.push(`/events/${idevent}`) }} href={`/events/${idevent}`}></button>
+    <button id="buttonDelete" type="button" onClick={() => deleteFormData(idtourist,idevent)}></button>  
   </div>
   )
 }
 
-export default DeleteOfBookingsByEvent;
-
-{/*export default class DeletionOfBookingsByTourist extends React.Component {
```
Comment (ghost) [inline front/src/component/DeletionOfBookingsByTourist.js:37]: So much dead code... it's a code necropolis !

**Comment by wasthishelpful** [comment]
conflicts to resolve :/

**Review by ghost — CHANGES_REQUESTED** [review]
Conflicts in three components

File: front/src/component/DeletionOfBookingsByTourist.js (hunk 2 above)
Comment (Cindie1908) [reply front/src/component/DeletionOfBookingsByTourist.js:37]: I didn't find dead code :( 
I'm not an archaeologist !

**Pull request merged**